src/routes/transactions/entities/confirmation-view/confirmation-view.entity.ts

```diff
@@ -8,10 +11,15 @@ interface Baseline {
 
 enum DecodedType {
   Generic = 'GENERIC',
+  CowSwapOrder = 'COW_SWAP_ORDER',
```

Comment:
Where this could potentially apply would be on the `TransactionInfoType` object which follows a different naming pattern. https://github.com/safe-global/safe-client-gateway/blob/a78fbf95630c348dcd411669eb4675761afae855/src/routes/transactions/entities/transaction-info.entity.ts#L4-L10

If the suggestion was to reuse it here or reuse this one for the `TransactionInfoType`, then we would be tying the types of this route with the types of `TransactionInfoType` (which can be completely different) 🤔

Reply:
I meant more where we map relative to CoW Swap-specific data. Thinking about it some more though, I guess we'd add extra cases if we were to introduce another swap-type. I'll resolve this.

Reply:
Oh I see what you mean 👍 Yeah good point. Although I think this feature (and all the properties that it introduces) are coupled to the CoW implementation.

In this case I think it makes sense that we do specify that it's specific to that service as this data might not be available or have no meaning in other services.

Probably we should be more specific in the `TransactionInfoType` though 🤔